c++/cpp2py/converters/pair.hpp

```diff
@@ -5,9 +5,9 @@ namespace cpp2py {
 
   template <typename T1, typename T2> struct py_converter<std::pair<T1, T2>> {
 
-    static PyObject *c2py(std::pair<T1, T2> const &p) {
-      pyref x1 = py_converter<T1>::c2py(std::get<0>(p));
-      pyref x2 = py_converter<T2>::c2py(std::get<1>(p));
+    static PyObject *c2py(std::pair<T1, T2> p) {
+      pyref x1 = py_converter<std::decay_t<T1>>::c2py(std::move(std::get<0>(p)));
```

Comment:
Same question:
I would have said : 
std::get<0>(std::move(p))
move p first, then using (4) in https://en.cppreference.com/w/cpp/utility/pair/get

Reply:
So with `std::get<1>(std::move(p))` we then use a `p` which has already been moved from? Isn't this something we want to avoid at all cost? Is this well-defined?

Reply:
Even if its safe this will most definitely trigger false-positives for static analyzers / warnings that warn about use after move!?